launch/sensors_launch.py

```diff
@@ -28,17 +29,21 @@ def generate_launch_description():
     SIMULATION = os.getenv('SIMULATION')
     simulation_mode = (SIMULATION == "1")
 
-    # DRONE_AIRFRAME = <t-drone | holybro | rover (same as holybro)>
-    is_robot_holybro_type = False
+    # DRONE_AIRFRAME = <holybro | rover (same as holybro) | m690 | m1200>
+    allowed_airframes = ["holybro", "rover", "m690", "m1200"]
     DRONE_AIRFRAME = os.getenv('DRONE_AIRFRAME')
-    if DRONE_AIRFRAME == "holybro" or DRONE_AIRFRAME == "rover":
-        is_robot_holybro_type = True
-    elif DRONE_AIRFRAME == "" or DRONE_AIRFRAME == "t-drone":
```

Comment:
This change comes with a deprecation as the t-drone (as airframe name) is not supported. For now, we can have an assumption that the t-drone actually means m690 by default. That way, it can still work with an older version of fog_hyper, while supporting the new m1200. So similar to rover where it launches the holybro file. In near future, this airframe setup will be replaced with common place to store/publish transformations.

Reply:
Solved.